pr-filter/package.json

```
@@ -0,0 +1,17 @@
+{
+  "name": "pr-filter",
```

minor (in case it isn't really used anywhere)
```suggestion
  "name": "@devextreme/pr-filter",
```

---

I've named it in the same way as other actions are (no scope):
- https://github.com/DevExpress/github-actions/blob/main/send-teams-notification/package.json#L2
- https://github.com/DevExpress/github-actions/blob/main/verify-version-change/package.json#L2

I suggest leave it as is for now